oak-commons/src/test/java/org/apache/jackrabbit/oak/commons/collections/MapUtilsTest.java

```diff
@@ -0,0 +1,325 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.jackrabbit.oak.commons.collections;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Objects;
+import java.util.Properties;
+import java.util.function.Predicate;
+
+/**
+ * Unit tests for the {@link MapUtils} class.
+ * <p>
+ * This class contains test cases to verify the functionality of the methods
+ * in the {@link MapUtils} class.
+ * </p>
```

Comment:
You don't need to close the \<p>. This is HTML, not XHTML. It even causes a javadoc warning.

Reply:
fixed in https://github.com/apache/jackrabbit-oak/pull/1989/commits/a79f797cdcb2b9fb7308fbba71e24aee2450b917